docs/release-notes/1.5.0/release-1.5.0.md

```diff
@@ -0,0 +1,79 @@
+# [ML.NET](http://dot.net/ml) 1.5.0
+
+## **New Features**
+- **New anomaly detection algorithm** ([#5135](https://github.com/dotnet/machinelearning/pull/5135)). ML.NET has previously supported anomaly detection through [DetectAnomalyBySrCnn](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.timeseriescatalog.detectanomalybysrcnn?view=ml-dotnet). This function operates in a streaming manner by computing anomalies around each arriving point and examining a window around it. Now we introduce a new function `DetectEntireAnomalyBySrCnn` that computes anomalies by considering the entire dataset and also supports the ability to set sensitivity and output margin. 
+- **Root Cause Detection** ([#4925](https://github.com/dotnet/machinelearning/pull/4925)) ML.NET now also supports root cause detection for anomalies detected in time series data. 
+
+## **Enhancements**
+- **Updates to TextLoader**
+  - Enable TextLoader to accept new lines in quoted fields  ([#5125](https://github.com/dotnet/machinelearning/pull/5125))
+  - Add escapeChar support to TextLoader ([#5147](https://github.com/dotnet/machinelearning/pull/5147))
+  - Add public generic methods to TextLoader catalog that accept Options objects ([#5134](https://github.com/dotnet/machinelearning/pull/5134))
+  - Added decimal marker option in TextLoader ([#5145](https://github.com/dotnet/machinelearning/pull/5145))
+- Onnxruntime updated to v1.3
```

Comment:
Perhaps add this PR as reference of updating Onnxruntime?
https://github.com/dotnet/machinelearning/pull/5104